contracts/HundredVesting.sol

```diff
@@ -2,11 +2,11 @@
 
 pragma solidity 0.8.3;
 
-import "@openzeppelin/contracts/access/Ownable.sol";
-import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
-contract HundredVesting is Ownable {
+contract HundredVesting {
+    using SafeERC20 for IERC20;
     IERC20 public immutable Hundred;
     uint256 totalPeriod = 365 * 24 * 60 * 60;
     uint256 timePerPeriod;
```

Comment:
I think `timePerPeriod` is not needed for calculating the remaining amount

Reply:
why? Did you try to test without it? It's failing without it.